### DISCUSSION TITLE
Unable to locate column config

### DISCUSSION THREAD

**Discussion by weixingp** [Budibase/budibase#4544] [Help] [answered]
I was trying to find out how to rename and reorder columns and saw column settings was added to v1.0.66-alpha.0
PR: https://github.com/Budibase/budibase/pull/4475
![153871116-fd199b92-2073-4a07-b08d-0c23e563045e](https://user-images.githubusercontent.com/72912936/154423686-44d3381d-9163-4ad4-b11c-86f0684eb99e.png)

I am using the latest docker image (should be v1.0.67) via docker compose but unable to locate this column config under table. 
Anyone know where this setting is?

**Comment by mjashanks** [comment, marked as answer]
Hey @weixingp - this feature is not due to be released until the first week in March.

**Reply by weixingp** [reply]
I see, thank you for your response. I guess i have to wait patiently then. Hope to see it live soon, very handy feature
